execution/executor/src/chunk_executor.rs

```diff
@@ -390,7 +390,7 @@ impl<V: VMExecutor> ChunkExecutorInner<V> {
                 txn_infos_with_proof,
             })?;
 
-        info!(
+        debug!(
```

Comment:
I feel we want to keep these 2 as info! as well.

Reply:
Done.